### ISSUE TITLE
Edge number problem of CiteSeer and Cora.

### ISSUE THREAD

**Issue by Villafly** [SeongJinAhn/VGNAE#5]
Hi,
When I ran your code, I found the edge numbers in the data instance are greater than edge number that you describled in Table 1. Could you please tell me what are the edges in the data.edge_index in detail? Thanks. As describled in the paper, you use a training set 60%, a validation set 10%, a test set 30% among all edges the same number of randomly sampled negative edges to the valid set and test set... But, the number is inconsistent with Table 1.

**Comment by SeongJinAhn** [comment]
Hi, weiweigu.

Thank you for finding mistakes.
I wrote the statistics based on the GCN paper. (https://arxiv.org/pdf/1609.02907.pdf)
But, I've compared every method with the dataset provided by PyG (torch geometric).
So, I'm convinced that my experiments were fair.
Sorry for your inconvenience.

**Comment by Villafly** [comment]
This issue is proposed here. FYI.
https://github.com/pyg-team/pytorch_geometric/issues/343
